web3.md

```diff
@@ -293,5 +307,38 @@ module WEB3
     rule <k> #evm_increaseTime => #sendResponse( "result" : Int2String(TS +Int DATA ) ) ... </k>
          <params> [ DATA:Int, .JSONList ] </params>
          <timestamp> ( TS:Int => ( TS +Int DATA ) ) </timestamp>
+
+    syntax KItem ::= "#eth_newBlockFilter"
+ // --------------------------------------
+    rule <k> #eth_newBlockFilter => #sendResponse ( "result": #unparseQuantity( FILTID )) ... </k>
+         <filters>
+           ( .Bag 
+          => <filter>
+               <filterID> FILTID </filterID>
+               <fromBlock> BLOCKNUM </fromBlock>
+               ...
+             </filter>
+           )
+           ...
+         </filters>
+         <number> BLOCKNUM </number>
+         <nextFilterSlot> ( FILTID:Int => FILTID +Int 1 ) </nextFilterSlot>
+
+    syntax KItem ::= "#eth_uninstallFilter"
+ // ---------------------------------------
+    rule <k> #eth_uninstallFilter ... </k>
+         <params> [ (DATA => #parseHexWord(DATA)), .JSONList ] </params>
+
+    rule <k> #eth_uninstallFilter => #sendResponse ( "result": "true" ) ... </k>
+         <params> [ FILTID, .JSONList ] </params>
+         <filters>
+           ( <filter>
+               <filterID> FILTID </filterID>
+               ...
+             </filter>
+          => .Bag
+           )
+           ...
+         </filters>
```

Comment:
Let's have an error case if it cannot find the given filter too, so that we don't accidentally halt.